### PR TITLE
Fix/whitelist otp

### PR DESCRIPTION
**Pull request by christabelneo**
This PR reenables SMS sending of OTP as well as OTP verification. Exceptions are included in both the generation and verification steps for the whitelisted contact number and OTP combination so that the SMS is not sent, and the OTP must be equal to the pre-determined value during verification.

N.B. also modified the seed data to use the team's numbers since SMS sending now works, to avoid sending test messages to random people

### CODE REVIEW

File: app/routes/otp.js [hunk 1]
```diff
@@ -106,6 +110,11 @@ async function generateOtp(req, res) {
       throw new OtpGenerationMalformedRequestError();
     }
 
+    // Whitelist contact for App Store review
+    if (contactNumber && contactNumber === WHITELIST_CONTACT.contactNumber) {
+      return res.status(200).send({ message: 'Whitelisted contact does not require new OTP' });
```
Comment (prestonlimlianjie) [inline app/routes/otp.js:115]: should we tell them that the OTP is 111111

File: app/src/seeders/20200403141358-dummy-orders.js [hunk 2]
```diff
@@ -2,12 +2,12 @@ module.exports = {
   up: async (queryInterface, Sequelize) => {
     const currentDatetime = new Date()
     let quarantineEnd = new Date()
-    quarantineEnd.setDate(quarantineEnd.getDate() + 14)
+    quarantineEnd.setDate(quarantineEnd.getDate() + 30)
```
Comment (kwajiehao) [inline app/src/seeders/20200403141358-dummy-orders.js:5]: Maybe we can set this longer?

File: app/routes/otp.js [hunk 3]
```diff
@@ -183,12 +192,20 @@ async function verifyOtp(req, res) {
...
-    //   return;
-    // }
+    if (contactNumber && contactNumber === WHITELIST_CONTACT.contactNumber) {
+      // 2. Check for whitelisted number and OTP combination
+      if (otp !== WHITELIST_CONTACT.otp) {
+        res.status(401).send('Invalid OTP');
+        return;
+      }
+    } else {
+      // 2. Retrieve the OTP
```
Comment (kwajiehao) [inline app/routes/otp.js:202]: Update the step number

File: app/routes/otp.js (hunk 1 above)
Comment (christabelneo) [reply app/routes/otp.js:115]: No this is out-of-band, we don't want real users to type in the number (accidentally or with malicious intent) and get the OTP

Comment (prestonlimlianjie) [reply app/routes/otp.js:115]: Maybe we then tell them to use the OTP that the Homer team has created

File: app/routes/otp.js (hunk 3 above)
Comment (christabelneo) [reply app/routes/otp.js]: resolved in 9375f57

File: app/src/seeders/20200403141358-dummy-orders.js (hunk 2 above)
Comment (christabelneo) [reply app/src/seeders/20200403141358-dummy-orders.js]: resolved in 23f02eb

File: app/routes/otp.js (hunk 1 above)
Comment (christabelneo) [reply app/routes/otp.js]: resolved in 5839e6b

**Pull request merged**